posts/faq/api-help.md

```diff
@@ -2,7 +2,15 @@
 title: How do I see the list of API functions I can call, and how do I call them?
 ---
 
-Here is an example script to get the documentation for the various API calls. To use any of the functions displayed, just provide any specified arguments in a dictionary.
+Here is an example script to get the documentation for the various API calls. To use any of the functions displayed, just provide any specified arguments in a dictionary. Many (though not all) of the calls are the same as those for bitcoin core, which are documented [here](https://en.bitcoin.it/wiki/Original_Bitcoin_client/API_calls_list).
+
+If for some reason you can't get lbrycrd-cli working to make these calls, make sure you passed -server when starting lbrycrd. If it still doesn't work, you can use this command if running lbrycrd with -server:
+
+`curl --user USER:PASSWORD --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "COMMAND", "params": [] }' -H 'content-type: text/plain;' http://127.0.0.1:9245/`
```

Comment:
Last nit: For long commands, its better to put them in a code block (using 4 indents) than to use the tick marks. Tick marks are best for short bits of code that are inline.